blueocean-bitbucket-pipeline/src/test/java/io/jenkins/blueocean/blueocean_bitbucket_pipeline/server/BitbucketServerEndpointTest.java

```diff
@@ -280,11 +281,66 @@ public void shouldFailOnIncompatibleVersion() throws UnirestException, IOExcepti
 
     }
 
+    // TODO: need a test case as unprivileged user
```

Comment:
I tested this manually and verified it produces a 403 as expected. Not sure how to configure a JUnit test with a more sophisticated security realm / access control, so I may just remove this TODO. I pinged @vivek but then remembered he is out this week. @michaelneale any thoughts?

Reply:
@cliffmeyers hrm maybe @imeredith may know. The JS tests run anon - so must be possible, but not sure if can turn of auth for a specific test (I thought there was some annotation...)

The other option is to have the test turn on anon read access via the settings.

Reply:
I'll drop @vivek an e-mail about this since he is out. I don't think it's critical at this point given that I tested manually.  (To clarify, this isn't ATH, it's just JUnit and I want to test a user without "superuser" level, which they all seem to have by default)